packages/nexus-sdk/src/Resource/__tests__/resource.spec.ts

```diff
@@ -17,19 +17,26 @@ describe('Resource', () => {
         'http://api.url/v1/resources/org/project/_/myId',
       );
       expect(fetchMock.mock.calls[0][1].method).toEqual('GET');
+      expect(fetchMock.mock.calls[0][1].headers).toEqual({
+        Accept: 'application/ld+json',
+      });
     });
 
     it('should make httpGet call to the resources api with the right url and query params', async () => {
       fetchMock.mockResponseOnce(JSON.stringify({ data: '' }));
-      await resource.get('org', 'project', 'myId', {
+      await resource.get<{ name: string }>('org', 'project', 'myId', {
         rev: 1,
         q: 'myTextString',
+        as: 'n-triples',
       });
       expect(fetchMock.mock.calls.length).toEqual(1);
       expect(fetchMock.mock.calls[0][0]).toEqual(
         'http://api.url/v1/resources/org/project/_/myId?rev=1&q=myTextString',
       );
       expect(fetchMock.mock.calls[0][1].method).toEqual('GET');
+      expect(fetchMock.mock.calls[0][1].headers).toEqual({
+        Accept: 'application/n-triples',
+      });
     });
   });
```

Comment:
Can we add a test for `vnd.graph-viz` (doesn't cost too much)

Reply:
added